x-pack/plugins/security/server/config.ts

```diff
@@ -149,6 +149,14 @@ export const ConfigSchema = schema.object({
   session: schema.object({
     idleTimeout: schema.nullable(schema.duration()),
     lifespan: schema.nullable(schema.duration()),
+    cleanupInterval: schema.duration({
+      defaultValue: '1h',
+      validate(value) {
+        if (value.asSeconds() < 60) {
```

Comment:
__question:__ I initially used `20m` as a minimum, but thought there may be cases for more aggressive cleanup. What do you think?